Replicator/Replicator.cc

```diff
@@ -268,7 +268,12 @@ namespace litecore::repl {
             _subRepls[coll].puller->start(_subRepls[coll].checkpointer->remoteMinSequence());
     }
 
-    pair<int, websocket::Headers> Replicator::httpResponse() const { return webSocket()->httpResponse(); }
+    pair<int, websocket::Headers> Replicator::httpResponse() const {
```

Comment:
_httpStatus/_httpHeaders are captured only when _onConnect is called. Is webSocket()->httpResponse() also captured when connection is made?

Reply:
Yes.